docs/spec.md

```diff
@@ -823,7 +823,7 @@ Afterwards, within each `process_group`:
 // %operand0@(1, 0): [5, 6, 7, 8]
 // %operand1@(0, 0): [9, 10, 11, 12]
 // %operand1@(1, 0): [13, 14, 15, 16]
-%result = "stablehlo.all_reduce"(%operand0, %operand0) ({
+%result:2 = "stablehlo.all_reduce"(%operand0, %operand0) ({
```

Comment:
Also change the examples in StableHLOOps.td

Reply:
StableHLOOps.td examples are non variadic, don't need the change.

Reply:
[Something to consider for a follow up]
Can we have the same example in both places as done for other ops?

Reply:
Let's keep the example same per 7.v in https://github.com/openxla/stablehlo/blob/main/docs/reference_checklist.md#after-implementing-the-op.

Reply:
https://github.com/openxla/stablehlo/pull/2481  will match the examples